modules/ROOT/pages/api-groups-modifying-groups.adoc

```diff
@@ -0,0 +1,115 @@
+= Modify API Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+If your business requirement changes, you might want to modify an API Group by adding new API instances to it. To cover new scenarios, you might want to add a new API instance. 
+
+You can also version an existing instance to add customizations. If a service is no longer required, you can remove the respective API instance that supports the service.
+
+The following tasks are covered in this section:
+
+* <<adding-api-instances-to-groups, Add API instances to Existing API Groups>>
+* <<removing-api-instances-from-groups, Remove API instances from an API Group>>
+* <<creating-new-versions-of-groups, Create new versions of an API Group>>
+
```

Comment:
I would remove all of this and make the previous text one paragraph.
And I would also add to the intro text that there are restrictions and how changes affect Exchange status.
The topic intro should refer to all content.

Reply:
Done! Thanks!